### PR TITLE
[Feature 9] Month Menu Button Reactions

### PR DESCRIPTION
**Pull request by YinLingithub**
Implemented the actionListener() method in MonthMenuV which calls add/edit Expense/Category views. There is also implementation of loadMonthMenu() in add/edit Expense/Category views, in order to update Main Menu when changes are made to expense list and category list in MonthlyStorage.

There are not tests for this pull request as changes are made in view classes.
Javadoc comments should be updated.

### CODE REVIEW

File: src/main/java/views/monthly_menu/MonthMenuV.java [hunk 1]
```diff
@@ -235,4 +255,16 @@ private static JTable getCategoryTable(Category[] categoryData){
         }
         return new JTable(categoryList, categoryTableTitle);
     }
+
+    private String[] getCategoryNames(){
```
Comment (katarinaavucic) [inline src/main/java/views/monthly_menu/MonthMenuV.java:301]: Missing a Javadoc here!

**Review by katarinaavucic — CHANGES_REQUESTED** [review]
Looks good so far just one missing Javadoc I found!

File: src/main/java/views/monthly_menu/MonthMenuV.java (hunk 1 above)
Comment (YinLingithub) [reply src/main/java/views/monthly_menu/MonthMenuV.java:301]: Ah, I didn't notice! Thanks!

**Review by luluwuwu — DISMISSED** [review]
approved

**Review by katarinaavucic — APPROVED** [review]
Looks good! YAYYY final (main) thing completed.

**Review by aricasas — APPROVED** [review]
All done!

**Review by luluwuwu — APPROVED** [review]
Good

**Pull request merged**